backend/src/main/resources/db/migration/V2__create_user.sql

```diff
@@ -1,7 +0,0 @@
-CREATE TABLE IF NOT EXISTS spar.user (
-  id         SERIAL,
-  email      VARCHAR(100) NOT NULL,
-  last_login TIMESTAMP NOT NULL DEFAULT NOW(),
-  CONSTRAINT user_pk
-    primary key(id)
-);
```

Comment:
We didn't change `UserEntity` at all, so it's still in our code base and associated with this now non-existing table. Shouldn't both be gone?

Reply:
Yeah, there was a commit missing. Sorry about that.